recipe/meta.yaml

```diff
@@ -11,22 +11,23 @@ source:
 build:
   number: 0
   script:
-    - {{ PYTHON }} -m pip install . -vv
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
     - rm "${SP_DIR}/pulp/solverdir/CoinMP.dll"  # [not win]
+  entry_points:
+    - pulptest = pulp.tests.run_tests:pulpTestAll
 
 requirements:
-  build:
-    - python                                 # [build_platform != target_platform]
-    - cross-python_{{ target_platform }}     # [build_platform != target_platform]
   host:
     - pip
     - python
+    - setuptools
+    - wheel
   run:
-    - amply >=0.1.2
-    # We use glpk on Windows because coincbc is only packaged on Unix, currently.
+    - python
+    - amply
```

Comment:
isn't amply an optional requirement for a plugin? I don't see it referenced in the code.

Reply:
Yeah, it seems it is redundant now. Thanks.